proto/recoverdatapb.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package recover_data;
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+// request to read region meata from a store
+message ReadRegionMetaRequest {
+    uint64 store_id = 1;
+}
+
+message RegionMeta {
+    uint64 region_id = 1;
+    uint64 peer_id = 2;
+    uint64 last_log_term = 3;
+    uint64 last_index = 4;
+    uint64 commit_index = 5;
+    uint64 version = 6;
+    bool tombstone = 7; //reserved, it may be used in late phase for peer check
+    bytes start_key = 8;
+    bytes end_key = 9;
+}
+
+// command to store for recovery data
+message RecoverCmdRequest {
+    uint64 region_id = 1;
+    bool as_leader = 2; // force region_id as leader
+    bool tombstone = 3; // set Peer to tombstoned in late phase
+}
+
+message RecoverCmdResponse {
```

Comment:
will this cmd failed? if so maybe add error reason field in Response? for easily debug in future.